WordPress/Classes/ViewRelated/Plans/PlanComparisonViewController.swift

```diff
@@ -33,12 +33,12 @@ class PlanComparisonViewController: PagedViewController {
     init(sitePricedPlans: SitePricedPlans, initialPlan: Plan, service: PlanService<StoreKitStore>) {
         self.siteID = sitePricedPlans.siteID
         self.pricedPlans = sitePricedPlans.availablePlans
-        self.activePlan = sitePricedPlans.activePlan
+        self.activePlan = sitePricedPlans.activePlan!
```

Comment:
He's right!

`!` should be considered dangerous; it says "please crash _here_"

Reply:
I think that's debatable, but this is probably not the time and place to discuss it 😛 

In any case, not force-unwrapping here requires quite a few changes in other places. I decided to not attack those changes to avoid making things even more complicated in this first iteration. I think we should attack this is a separate issue / PR. What do you think?

Reply:
(just to clarify, what I find debatable are blanket statements like "crashing is always bad". Sometimes I think crashing is actually better than letting a system work in an inconsistent state)

Reply:
A separate PR works, tho it feels like something we may not get back around too 😉

I prefer to explicitly crash with a `fatalError` when it’s preferrable to crash, since it makes it obvious a decision was made and not just the more common “just work for now please” 😂

Anyway, this is just a comment and not a proper review because I don’t want to hold up the PR with my holy war